core/types/block.go

```diff
@@ -40,6 +40,40 @@ var (
 	EmptyUncleHash = rlpHash([]*Header(nil))
 )
 
+type VerifyStatus struct {
```

Comment:
I think it is better to create a new golang file than reuse `block.go`.

Reply:
You are right, we plan to move these definition into verify manager related file, here is temporary solution.